operator/docs/examples/dataplane.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kit.k8s.sh/v1alpha1
+kind: DataPlane
+metadata:
+  name: example-nodes 
+spec: 
+  clusterName: example # Desired Cluster Name
```

Comment:
Thoughts on making this fully decoupled from the custom resource and doing something like

```
replicas: 5 # instead of nodecount, node is implied by the fact that this is a dataplane.
cluster:
  name: ...
  endpoint: ...
```